helm/kubernetes-test-app-chart/templates/rbac.yaml

```diff
@@ -12,9 +12,6 @@ subjects:
 roleRef:
   kind: ClusterRole
   name: {{ .Values.name }}
-  labels:
-    app: {{ .Values.name }}
-    giantswarm.io/service-type: "managed"
```

Comment:
You can't have labels here because it's a role ref. Helm 3 is stricter and doesn't allow the chart to be installed.